emails/models.py

```diff
@@ -200,9 +200,8 @@ def __str__(self):
     def make_domain_address(user, address=None):
         if address is None:
             address = address_default()
-        domain_address = DomainAddress.objects.create(user=user, address=address)
         address_contains_badword = any(
-            badword in domain_address.address
+            badword in address
             for badword in emails_config.badwords
         )
```

Comment:
This might be too restrictive for domain addresses. The [`badwords.txt`](https://github.com/mozilla/fx-private-relay/blob/main/emails/badwords.txt) is very broad and meant to prevent *accidental* use of some words by randomly-generated text. There are things in there like `bi`, but someone who signs up for Business Insider might try to use `bi@subdomain.domain.com` and be confused why it's considered bad.

Reply:
I agree and I would like more product decision on what we should allow DomainAddress can do.
1. Check that subdomain does not contain some/specific bad words?
2. Check that randomly generated aliases via RelayAddress or DomainAddress get checked for `badwords.txt`?
3. Check that alias not randomly generated contain some/specifc bad words?
Of these three situations we may want to check for badwords what do you think should be a product decision?

Reply:
1. Yes, prevent `badwords`
2. Yes, prevent `badwords`
3. No, don't prevent `badwords`

Reply:
Resolved with https://github.com/mozilla/fx-private-relay/pull/812/commits/1afd316e6dc4184b7bcb819978848e2cb63d199b